examples/dt/uni01alpha/control-plane/service-values.yaml

```diff
@@ -54,8 +54,9 @@ data:
   swift:
     enabled: true
 
+  # ToDo: octavia needs to be enabled once the issue is fixed.
```

Comment:
It would be good to link to the issue that is requiring fixing to know when this should be re-enabled. A task could also be added to the linked issue tracking the fix so that this can be re-enabled at the same time.